utils/utils.js

```diff
@@ -63,6 +63,7 @@ async function getTree(
 // send the new tree object back to Github and point the latest commit on the staging branch to it
 async function sendTree(
```

Comment:
can change this to an object and use destructuring? else its not very clear when we miss out on a parameter

Reply:
Discussed with @alexanderleegs, since all these `classes/` files and this `utils/getTree`, `utils/sendTree` function will be removed by the refactor, it might not be worth the effort to change all the classes to use the new object parameter. 

In place, we will just test extensively.